src/main/resources/anet-schema.yml

```diff
@@ -94,6 +94,15 @@ definitions:
             params:
               type: array
           required: [type]
+      aggregation:
+        type: object
+        properties:
+          widget:
+            type: string
+            enum: [likertScale, numberAggregation, default]
+          aggregationType:
+            type: string
+            enum: [sum, avg, min, max]
```

Comment:
These apply only to `numberAggregation`.

Reply:
Changed the schema.